### PR TITLE
fix bad url in how-to-doc.md

### PR DESCRIPTION
**Pull request by wackxu**
fix bad url

### CODE REVIEW

File: contributors/devel/how-to-doc.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Use inline links instead of urls at all times. When you add internal links to
 `http://releases.k8s.io/HEAD/<path/to/link>`. For example, avoid using:
 
 ```
-[GCE](https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/gce.md)  # note that it's under docs/
+[GCE](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/getting-started-guides/gce.md)  # note that it's under docs/
```
Comment (xiangpengzhao) [inline contributors/devel/how-to-doc.md]: We may need to edit the example instead of just the link.

Comment (cblecker) [inline contributors/devel/how-to-doc.md]: Link should be https://kubernetes.io/docs/getting-started-guides/gce/

Comment (cblecker) [reply contributors/devel/how-to-doc.md]: ping @wackxu

Comment (wackxu) [reply contributors/devel/how-to-doc.md]: @cblecker Thanks for your kindly suggest.And I have read the file again.I agree with the opinion of @xiangpengzhao ,We should also edit the example and I have fix it.

**Comment by cblecker** [comment]
please squash :)

**Comment by wackxu** [comment]
@cblecker I have fix it, thanks!

**Pull request merged**